app/views/components/link.scala.html

```diff
@@ -14,11 +14,12 @@
  * limitations under the License.
  *@
 
-@import models.ui.HyperLink
+@import models.ui.Link
 
-@(link: HyperLink)(implicit messages: Messages)
+@(link: Link)(implicit messages: Messages)
```

Comment:
Do we still require the implicit messages here? Looking at render_text I suspect we will have either an english or welsh link passed into this component.

Reply:
No we dont, I will remove